.github/workflows/main.yml

```diff
@@ -61,12 +94,21 @@ jobs:
       if: github.event_name == 'release'
       run: |
         bash ./dist/osx/build.sh dmg
-    - name: Upload to GitHub Release
+    - name: Update Appcast
       if: github.event_name == 'release'
       run: |
-        TAG_NAME=$(./dist/get-tag-name.sh)
-        ./dist/upload-github-release-asset.sh github_api_token=${{ secrets.GITHUB_TOKEN }} owner=MartinBriza repo=toggldesktop tag="$TAG_NAME" filename="TogglDesktop.dmg" renameto="TogglDesktop-${TAG_NAME/v/}.dmg"
-
+        base64 -D <<< "${{ secrets.APPCAST_PRIVATE_PEM }}" > dsa_priv.pem
+        bash ./dist/osx/build.sh appcast
+    - name: Upload
+      if: github.event_name == 'release'
+      run: |
+        export "GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}"
+        bash ./dist/osx/build.sh upload
```

Comment:
I would prefer to reuse as possible rather than tweaking the current script, which support various platforms (They're already working). I would open separate ticket it for it